.eslintrc.json

```diff
@@ -4,6 +4,9 @@
     "ecmaVersion": 12
   },
   "rules": {
+    /* To work towards no-build. */
+    "import/extensions": ["warn", "always", { "js": "always" }],
```

Comment:
Webpack doesn't need the `.js` suffix, but if the goal is to work towards a no-build setup it will be required. Thought it would be a good idea to start enforcing it regardless.